model/user.go

```diff
@@ -15,7 +15,6 @@ import (
 const (
 	ROLE_ADMIN           = "admin"
```

Comment:
I will rename the var name to ROLE_TEAM_ADMIN, but the value will still be "admin"